### PR TITLE
Closes ImportBot importing non-covers (For Amazon Imports)

### LINKED ISSUE #6089: ImportBot importing non-covers
Sometimes ImportBot imports placeholder covers. Can we check the url before importing the cover? These likely are all the same url.

### Evidence / Screenshot (if possible)
image

### Relevant url?
https://openlibrary.org/works/OL27276169W/The_Modern_Presidency

### Stakeholders
@mekarpeles

### PR DESCRIPTION
**Pull request by b2pacific**
Closes #6089 

hotfix

### Technical

Currently added the file name of the placeholder image that we could find online, the filename might change in the future.

### Stakeholders

@jimchamp @mekarpeles

### CODE REVIEW

**Comment by mekarpeles** [comment]
This LGTM @jimchamp we may want to patch deploy and manually try adding on testing before merging

**Comment by tuminzee** [comment]
Hi @b2pacific I updated it with a more meaningful name.

Let's use self-documenting titles for issues and pull requests. When looking at a list like:
https://github.com/internetarchive/openlibrary/pulls

**Comment by mekarpeles** [comment]
@cdrini + @jimchamp confirmed that the image key of the work coming through our API is:

```
[{'url': 'https://www.amazon.com/dp/0231206666/?tag=internetarchi-20', 'source_records': ['amazon:0231206666'], 'isbn_10': ['0231206666'], 'isbn_13': ['9780231206662'], 'price': '$90.00', 'price_amt': 9000, 'title': 'The Modern Presidency: Six Debates That Define the Institution', 'cover': 'https://m.media-amazon.com/images/I/01RmK+J4pJL._SL500_.gif', 'authors': [{'name': 'Genovese, Michael'}], 'publishers': ['Columbia University Press'], 'number_of_pages': 160, 'edition_num': None, 'publish_date': 'Sep 20, 2022', 'product_group': 'Book', 'physical_format': 'hardcover'}]
```

which matches `01RmK+J4pJL`

We also confirmed with `9786057834683`

File: openlibrary/core/vendors.py [hunk 1]
```diff
@@ -241,12 +241,7 @@ def serialize(product):
                 and item_info.title
                 and getattr(item_info.title, 'display_value')
             ),
-            'cover': (
-                images
-                and images.primary
-                and images.primary.large
-                and images.primary.large.url
-            ),
+            'covers': (images.primary.large.url if images and images.primary and images.primary.large and images.primary.large.url and not '01RmK+J4pJL.' in images.primary.large.url else None),
```
Comment (mekarpeles) [inline openlibrary/core/vendors.py:244]:
```suggestion
            'covers': (images.primary.large.url if images and images.primary and images.primary.large and images.primary.large.url and not '/01RmK+J4pJL.' in images.primary.large.url else None),
```

**Review by jimchamp — APPROVED** [review]
Lgtm, thanks @b2pacific!

**Pull request merged**